src/oss/langgraph/workflows-agents.mdx

```diff
@@ -0,0 +1,2221 @@
+});
+
+// Route the input to the appropriate node
+const llmCallRouter = task("router", async (input: string) => {
+  const decision = await router.invoke([
+    {
+      role: "system",
+      content: "Route the input to story, joke, or poem based on the user's request."
+    },
+    {
+      role: "user",
+      content: input
+    },
+  ]);
+  return decision.step;
+});
+
+// Build workflow
+const workflow = entrypoint(
+  "routerWorkflow",
+  async (input: string) => {
+    const nextStep = await llmCallRouter(input);
+
+    let llmCall;
+    if (nextStep === "story") {
+      llmCall = llmCall1;
+    } else if (nextStep === "joke") {
+      llmCall = llmCall2;
+    } else if (nextStep === "poem") {
+      llmCall = llmCall3;
+    }
+
+    const finalResult = await llmCall(input);
+    return finalResult;
+  }
+);
+
+// Invoke
+const stream = await workflow.stream("Write me a joke about cats", {
+  streamMode: "updates",
+});
+
+for await (const step of stream) {
+  console.log(step);
+}
+```
+</Tab>
+</Tabs>
+:::
+
+## Orchestrator-worker
+
+In an orchestrator-worker configuration, the orchestrator:
+
+- Breaks down tasks into subtasks
+- Delegates subtasks to workers
+- Synthesizes worker outputs into a final result
+
+![worker.png](/oss/images/worker.png)
+
+Orchestrator-worker configurations provide more flexibility and are often used when subtasks can't be predefined the way they can with [parallelization](#parallelization). This is common with workflows that write code or need to update content across multiple files. For example, a workflow that needs to update installation instructions for multiple Python libraries across an unknown number of documents might use this pattern.
+
+### Code examples
+
+:::python
+<Tabs>
+<Tab title="Graph API">
+```python
+from typing import Annotated, List
+import operator
+
+
+# Schema for structured output to use in planning
+class Section(BaseModel):
+    name: str = Field(
+        description="Name for this section of the report.",
+    )
+    description: str = Field(
+        description="Brief overview of the main topics and concepts to be covered in this section.",
+    )
+
+
+class Sections(BaseModel):
+    sections: List[Section] = Field(
+        description="Sections of the report.",
+    )
+
+
+# Augment the LLM with schema for structured output
+planner = llm.with_structured_output(Sections)
+```
+
+### Creating workers in LangGraph
+
+Orchestrator-worker workflows are common and LangGraph has built-in support for them. The `Send` API lets you dynamically create worker nodes and send them specific inputs. Each worker has its own state, and all worker outputs are written to a shared state key that is accessible to the orchestrator graph. This gives the orchestrator access to all worker output and allows it to synthesize them into a final output. The example below iterates over a list of sections and uses the `Send` API to send a section to each worker.
+
+```python
```

Comment:
Missing Graph API tag? And python vs js fencing? Or is this only available in python?